.github/actions/devcontainer_run_command/action.yml

```diff
@@ -0,0 +1,213 @@
+      description: "The friendly name of the command."
+      required: true
+    ACTIONS_ACR_NAME:
+      description: "The Azure Container registry name that the devcontainer is stored in."
+      required: true
+    ACTIONS_ACR_PASSWORD:
+      description: "The Azure Container registry password for the devcontainer."
+      required: true
+    ARM_TENANT_ID:
+      description: "Azure Tenant Id."
+      required: true
+    ARM_CLIENT_ID:
+      description: "Azure user Id. This user needs User Access Administrator permission as minimum."
+      required: true
+    ARM_CLIENT_SECRET:
+      description: "Azure user password."
+      required: true
+    ARM_SUBSCRIPTION_ID:
+      description: "Azure Subscription Id."
+      required: true
+    RESOURCE_LOCATION:
+      description: "The Azure Region (e.g. WestEurope)."
+      required: true
+    RESOURCE:
+      description: "The API Client Id."
+      required: true
+    AUTH_TENANT_ID:
+      description: "The Tenant Id where the App is registered."
+      required: true
+    TEST_APP_ID:
+      description: "The Test Application Id used to interact with the API."
+      required: true
+    TEST_USER_NAME:
+      description: "The Test User used to interact with the API."
+      required: true
+    TEST_USER_PASSWORD:
+      description: "The Test User password Id used to interact with the API."
+      required: true
+    TEST_WORKSPACE_APP_ID:
+      description: "The Test Workspace Id used to interact with the API."
+      required: true
+    TRE_ID:
+      description: "The TRE Id."
+      required: true
+    TF_VAR_location:
+      description: "The Azure Region (e.g. WestEurope)."
+      required: true
+    TF_VAR_tre_id:
+      description: "The TRE Id."
+      required: true
+    TF_VAR_terraform_state_container_name:
+      description: "The name of the container to store the Terraform state."
+      required: true
+    TF_VAR_mgmt_resource_group_name:
+      description: "The resource group used to store the Terraform state."
+      required: true
+    TF_VAR_mgmt_storage_account_name:
+      description: "The storage account used to store the Terraform state."
+      required: true
+    TF_VAR_core_address_space:
+      description: "Core address space."
+      required: true
+    TF_VAR_tre_address_space:
+      description: "TRE address apace."
+      required: true
+    TF_VAR_swagger_ui_client_id:
+      description: "The Swagger UI Client ID."
+      required: true
+    TF_VAR_aad_tenant_id:
+      description: "The Tenant Id where the App is registered. (Same as AUTH_TENANT_ID)"
+      required: true
+    TF_VAR_api_client_id:
+      description: "The API Client Id. (Same as Resource)"
+      required: true
+    TF_VAR_api_client_secret:
+      description: "The API Client Secret."
+      required: true
+    TF_VAR_api_app_service_plan_sku_tier:
+      description: "The SKU tier of the App Service for the API."
+      required: false
+      default: "PremiumV3"
```

Comment:
This is a duplicate of the default value defined in TF (also next var). Do you have to specify a value here? What would happen if you don't?

Reply:
Perfect - they were originally in the workflow.yml, so replicated. Never checked the default in Terraform - They have now gone as I can't see us changing them in the pipeline